packages/components/src/molecules/ProductTitle/ProductTitle.tsx

```diff
@@ -22,20 +25,45 @@ export type ProductTitleProps = Omit<HTMLAttributes<HTMLElement>, 'title'> & {
    * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
    */
   testId?: string
-}
+  /**
+   * The current value of the rating, based on the quantity of child elements.
+   */
+  ratingValue?: number
```

Comment:
This description confused me a bit. This prop will be a number passed from the parent to this component, and that value is going to be passed as it was received to `Rating`, right? How is it related to the number of child elements?

Reply:
hmmm, you are right! I was not not sure the best way to pass this prop though, we need to pass a number value to `Rating` component, and the default child length is 5 (number of stars to be filled). Maybe I was wrong and we should also pass `Rating's` length prop here. 🤔

Reply:
We will leave 5 as the default value :)